packages/replay/metadata/test.js

```diff
@@ -0,0 +1,43 @@
+const {
+  array,
+  create,
+  defaulted,
+  enums,
+  number,
+  object,
+  optional,
+  string,
+} = require("superstruct");
+
+const VERSION = 1;
+
+const versions = {
+  1: object({
+    version: defaulted(number(), () => 1),
+    title: string(),
+    result: enums(["passed", "failed", "timedOut"]),
+    path: optional(array(string())),
+    run: optional(string()),
+    file: optional(string()),
+  }),
+};
+
+function validate(metadata) {
+  return init(metadata && metadata.test);
```

Comment:
Maybe we should put a guard here rather than doing the `&&`? I could see that throwing some cryptic error on line 30 like `false does not have .version` or whatever, when the real problem is that `metadata` was falsey.